Document MMS (Multimedia Messaging) support (#761)
mautic/user-documentation: docs/channels/sms.rst

**Unpublish at (date/time)** - This allows you to define the date and time at which this Text Message ceases to be available for sending to Contacts.

.. vale off

> Sending MMS (Multimedia Messages)

Author: Added MMS documentation based on [PR #15999](https://github.com/mautic/mautic/pull/15999) which implements MMS support for Text Messages. The PR's `translations/messages.en.ini` file specifies regional availability (US/Canada only), media limits (10 images, 5 MB total), and supported formats (.gif, .png, .jpg, .jpeg).

Source: https://github.com/mautic/mautic/pull/15999

> =================================

Reviewer: ```suggestion
.. vale off

Sending Multimedia Messages - MMS
=================================

.. vale on
```

.. vale on

> Mautic supports sending MMS, which allows you to attach images to your Text Messages. MMS messages appear in the Text Messages list with an image icon indicator, and the Campaign Builder dropdown shows an ``[MMS]`` prefix for messages with media attached.

Reviewer: Add MMS to `accept.txt` file.

.. note::

>     MMS is currently available for Contacts with phone numbers in the United States, Canada, and Australia only. When sending to Contacts outside these regions, Mautic sends the image as a URL link instead.

Author: Added Australia to the MMS regional availability list based on the PR description which states "MMS via Twilio work only in US, Canada and Australia."

Source: https://github.com/mautic/mautic/pull/15999

>     MMS is currently available for Contacts with phone numbers in the United States, Canada, and Australia only. When sending to Contacts outside these regions, Mautic sends the image as a URL link instead.

Reviewer: ```suggestion
.. note::
   
   MMS is currently available for Contacts with phone numbers in the United States, Canada, and Australia only. When sending to Contacts outside these regions, Mautic sends the image as a URL link instead.
```

.. vale off

Enabling MMS for a Text Message
-------------------------------

.. vale on

To send a Text Message with media:

1. Create or edit a Text Message.

2. Toggle the **Is MMS** option to enable media attachments.

3. Add media using one of the following methods:

   - Click the **Upload Media** button to select an image from your Mautic Asset Manager
   - Enter an external image URL in the text field and press Enter or click the arrow button to add it

> 4. To remove an attached image, click the **X** overlay that appears when you select the image thumbnail.

Reviewer: Follow our style guide to work with lists in RST.

```suggestion
#. Create or edit a Text Message.
#. Toggle the **Is MMS** option to enable media attachments.
#. Add media using one of the following methods:

   * Click the **Upload Media** button to select an image from your Mautic Asset Manager
   * Enter an external image URL in the text field and press Enter or click the arrow button to add it

#. To remove an attached image, click the **X** overlay that appears when you select the image thumbnail.
```

.. vale off

MMS media requirements
----------------------

.. vale on

- **Supported formats**: ``.gif``, ``.png``, ``.jpg``, and ``.jpeg``
- **Maximum images**: 10 images per MMS
- **Total size limit**: 5 MB for all attached media combined

.. note::

>     Mautic doesn't validate external URLs. Ensure your external images use a supported format and the total size is under 5 MB.

Reviewer: ```suggestion
.. note::
   
   Mautic doesn't validate external URLs. Ensure your external images use a supported format and the total size is under 5 MB.
```

**Unpublish at (date/time)** - This allows you to define the date and time at which this Text Message ceases to be available for sending to Contacts.

.. vale off

.. _sending MMS in SMS section:

Sending Multimedia Messages - MMS
=================================

.. vale on

Mautic supports sending MMS, which allows you to attach images to your Text Messages. MMS messages appear in the Text Messages list with an image icon indicator, and the Campaign Builder dropdown shows an ``[MMS]`` prefix for messages with media attached.

.. note::

>     MMS is currently available for Contacts with phone numbers in the United States, Canada, and Australia only. When sending to Contacts outside these regions, Mautic sends the image as a URL link instead.

Reviewer: ```suggestion
.. note::
   
   MMS is currently available for Contacts with phone numbers in the United States, Canada, and Australia only. When sending to Contacts outside these regions, Mautic sends the image as a URL link instead.
```
